protocol/x/prices/keeper/update_price.go

```diff
@@ -74,22 +74,9 @@ func (k Keeper) GetValidMarketPriceUpdates(
 		// Skip proposal logic in the event of invalid inputs, which is only likely to occur around network genesis.
 		if !indexPriceExists {
 			metrics.IncrCountMetricWithLabels(types.ModuleName, metrics.IndexPriceDoesNotExist, marketMetricsLabel)
-			// Conditionally log missing index prices at least 20s after genesis/restart/market creation. We expect that
-			// there will be a delay in populating index prices after network genesis or a network restart, or when a
-			// market is created, it takes the daemon some time to warm up.
-			if !k.IsRecentlyAvailable(ctx, marketId) {
-				nonExistentMarkets = append(nonExistentMarkets, marketId)
-			}
+			nonExistentMarkets = append(nonExistentMarkets, marketId)
 			continue
 		}
-		if len(nonExistentMarkets) > 0 {
```

Comment:
This code block should not be within this for loop, which just results in logging out this error in every iteration afterwards, even if just once market is missing index price (see example error [log](https://app.datadoghq.com/logs?query=environment%3A%28staging%20OR%20mainnet%20OR%20testnet%29%20%22Index%20price%20for%20markets%20does%20not%20exist%22%20&agg_m=count&agg_m_source=base&agg_t=count&cols=host%2Cservice&fromUser=true&messageDisplay=inline&refresh_mode=paused&storage=hot&stream_sort=desc&viz=stream&from_ts=1719439365000&to_ts=1720044165000&live=false)). Moved logging to after the loop.

Reply:
Removed `MarketIsRecentDuration`. `PriceDaemonInitializationBlocks` is still used in some other legacy logging logic, which we can deprecate in a future PR.